spec/requests/core_induction_programme/course_year_spec.rb

```diff
@@ -22,15 +22,15 @@
 
     describe "GET /years/new" do
       it "renders the cip new years page" do
-        get "/years/new"
+        get "/#{cip.to_param}/new"
         expect(response).to render_template(:new)
       end
     end
 
-    describe "POST /years" do
+    xdescribe "POST /years" do
```

Comment:
the `xdescribe`s in this file are intentional - i'm not sure why the tests used to work, but the functionality is broken when access through a browser so they shouldn't have been. needs fixing in a separate PR (and these tests bringing back with it)

Reply:
Could you make a JIRA ticket for fixing it and leave a code comment linking to it?

Reply:
done! https://dfedigital.atlassian.net/browse/CPDEL-649